net/zmap/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           github 1.0
+
+name                zmap
+github.setup        zmap zmap 3.0.0-beta1 v
+
+categories          net
+platforms           darwin
+license             apache-2
+maintainers         {@artkiver gmail.com:artkiver} openmaintainer
+
+description         ZMap is a fast network scanner
+long_description    designed for Internet-wide network surveys capable of \
+                    scanning the entire IPv4 address space in 5 minutes from \
+                    a 10GbE connection.
+homepage            https://zmap.io/
+
+checksums           rmd160  4b30c0f33c5e4503a89dcb37808c50075373c4f4 \
+                    sha256  61787bd59cb23183a8b5424a148b95812d8254d6249c2d4efb31cbebff9ca383 \
+                    size    154851
+
+depends_build       port:byacc \
```

Comment:
Since you're using portgroups, it's always best to use the `-append` form for `depends_*` specifications, just to ensure you're not overwriting depend specifications that are already being set by any included portgroups.

- `depends_build` -> `depends_build-append`
- `depends_lib` -> `depends_lib-append`

Reply:
Ah, thank you for the clarification. 

https://trac.macports.org/ticket/42895#comment:3 mentioned something similar, but the nuance was lost on me.